var/spack/repos/builtin/packages/ninja/package.py

```diff
@@ -69,3 +69,9 @@ def install(self, spec, prefix):
         # instead of 'ninja'. Install both for uniformity.
         with working_dir(prefix.bin):
             symlink("ninja", "ninja-build")
+
+    def setup_dependent_package(self, module, dspec):
+        name = "ninja"
+        if sys.platform == "win32":
+            name = name + ".exe"
+        module.ninja = which(name, path=[self.spec.prefix.bin], required=True)
```

Comment:
Should we make `which` just automatically try with/without `.exe` on windows?

Reply:
That's a really good point, I'll check that and add it if we aren't doing it.

Reply:
Apparently we are, so I'm removing this check.